.projectile

```diff
@@ -0,0 +1 @@
+- *.pyc
```

Comment:
Another useful emacs file, but I can remove it if people think it shouldn't be here.